pkg/adaptor/hypervisor/vsphere/types.go

```diff
@@ -10,7 +10,7 @@ type Config struct {
 	VcenterURL   string
 	UserName     string
 	Password     string
-	Insecure     bool
```

Comment:
Sorry if it's obvious, could you explain this what is it and why it is replaced with thumbprint

Reply:
Insecure = false is not implemented and is true by default. Its also not mentioned in the blog. If you want secure mode you have to provide a thumbprint. So instead of having 2 parameters we only need one. If Thumbprint is set then you want secure mode.